api/projects/permissions.py

```diff
@@ -41,6 +41,16 @@ def has_permission(self, request, view):
             organisation = Organisation.objects.get(
                 id=int(request.data.get("organisation"))
             )
+
+            # Allow project creation based on the active subscription
+            subscription_metadata = (
+                organisation.subscription.get_subscription_metadata()
+            )
+            total_projects_created = Project.objects.filter(
+                organisation=organisation
+            ).count()
+            if total_projects_created >= subscription_metadata.projects:
```

Comment:
I think `subscription_metadata.projects` can be (and is by default) `None` which indicates no limit on projects. I think we should therefore replace this with

```suggestion
            if subscription_metadata.projects and total_projects_created >= subscription_metadata.projects:
```